StackSplit/SS_stacksplit_start.m

```diff
@@ -435,12 +435,12 @@ function call_pushEXIT(hObject,~,~)
 
 if exit==1
 
-    choice = questdlg('Really want to exit?','Exit menu','Yes','No','Yes');
+    choice = questdlg('Do you really want to exit?','Exit menu','Yes','No','Yes');
 
     switch choice
     case 'Yes'
 
-        h.fig
+        h.fig;
```

Comment:
Currently I do not see why this line is needed. Thus I suggest to remove it completely, instead of only suppressing the output.
```suggestion
```